kernel/include/devices/Rtc.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#define CMOS_ADDRESS_REGISTER   0x70
+#define CMOS_DATA_REGISTER  0x71
+#define BASE_CENTURY 2000
+#define SECONDS_IN_A_DAY 86400
+namespace Kernel::Devices
+{
+
+    enum Rtc_Registers : uint8_t 
+    {
+        Seconds = 0x00,
+        Minutes = 0x02,
+        Hours = 0x04,
+        WeekDay = 0x06,
+        DayOfMonth = 0x07,
+        Month = 0x08,
+        Year = 0x09
+
+    };
+
+    uint8_t readRtcRegister(uint8_t rtcRegister);
```

Comment:
Since you've defined an enum for RtcRegisters, why not make that the argument here instead of a uint8_t? That would only allow valid registers to be passed in args then.